packages/@atjson/source-html/test/converter-test.ts

```diff
@@ -952,5 +952,60 @@ describe("@atjson/source-html", () => {
         ],
       });
     });
+    test("Reddit Embed code", () => {
+      let doc = HTMLSource.fromRaw(
+        `<iframe id="reddit-embed"
+        src="https://www.redditmedia.com/r/IndianDankMemes/comments/qlndlm/average_indian_family/?ref_source=embed&amp;ref=share&amp;embed=true"
+        sandbox="allow-scripts allow-same-origin allow-popups"
+        style="border: none;"
+        height="476"
+        width="640"
+        scrolling="no"></iframe>`
+      ).convertTo(OffsetSource);
+
+      let hir = new HIR(doc).toJSON();
+      expect(hir).toMatchObject({
+        type: "root",
+        children: [
+          {
+            type: "iframe-embed",
+            attributes: {
+              url: "https://www.redditmedia.com/r/IndianDankMemes/comments/qlndlm/average_indian_family/?ref_source=embed&ref=share&embed=true&showmedia=false",
+              height: "476",
+              width: "640",
+            },
+            children: [],
+          },
+        ],
+      });
```

Comment:
```suggestion
      expect([...doc.where({ type: "-offset-iframe-embed" })]).toMatchObject([
        {
          type: "iframe-embed",
          attributes: {
            url: "https://www.redditmedia.com/r/IndianDankMemes/comments/qlndlm/average_indian_family/?ref_source=embed&ref=share&embed=true&showmedia=false",
            height: "476",
            width: "640",
          },
        }]
      );
```

Reply:
Testing against the HIR is a legacy type of test and we'd prefer tests to be done against the atjson document directly.